sfputil/main.py

```diff
@@ -1968,10 +1968,12 @@ def debug():
 # 'loopback' subcommand
 @debug.command()
 @click.argument('port_name', required=True, default=None)
-@click.argument('loopback_mode', required=True, default="none",
-                type=click.Choice(["none", "host-side-input", "host-side-output",
+@click.argument('loopback_mode', required=True, default=None,
+                type=click.Choice(["host-side-input", "host-side-output",
                                    "media-side-input", "media-side-output"]))
-def loopback(port_name, loopback_mode):
+@click.argument('enable', required=True, default=None,
```

Comment:
@xinyulin Why do we need `default` here since we already have `required` set to True.

Reply:
@mihirpat1 Thanks! I just realized that `default` is unnecessary if `required` is set to True. All defaults were removed in the latest update.